### PR TITLE
Add Session UI

### PR DESCRIPTION
**Pull request by chuayijing**

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -35,14 +35,25 @@
...
+    private SessionListPanel sessionListPanel;
+
     @FXML
     private StackPane commandBoxPlaceholder;
 
     @FXML
     private MenuItem helpMenuItem;
 
     @FXML
-    private StackPane personListPanelPlaceholder;
```
Comment (chuayijing) [inline src/main/java/seedu/address/ui/MainWindow.java:45]: Little unsure of this part, might need @PotatoCombat help on fxml

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -108,7 +119,11 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      */
     void fillInnerParts() {
         personListPanel = new PersonListPanel(logic.getFilteredPersonList());
-        personListPanelPlaceholder.getChildren().add(personListPanel.getRoot());
+        studentListPanelPlaceholder.getChildren().add(new PersonListPanel(logic.getFilteredPersonList()).getRoot());
+        moduleListPanelPlaceholder.getChildren().add(new PersonListPanel(logic.getFilteredPersonList()).getRoot());
+        groupListPanelPlaceholder.getChildren().add(new PersonListPanel(logic.getFilteredPersonList()).getRoot());
+        sessionListPanel = new SessionListPanel(logic.getFilteredSessionList());
```
Comment (chuayijing) [inline src/main/java/seedu/address/ui/MainWindow.java:125]: also a little unsure of the fxml this part, might need @PotatoCombat help

File: src/main/java/seedu/address/ui/SessionCard.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    private Label date;
+    @FXML
+    private Label description;
+
+    public SessionCard(Session session, int displayedIndex) {
+        super(FXML);
+        this.session = session;
+        id.setText(displayedIndex + ". ");
+        // dont really know how to convert enum to string?? ****
+        sessionType.setText(session.getSessionType().SESSION_TYPE_CONSULTATION.toString());
```
Comment (chuayijing) [inline src/main/java/seedu/address/ui/SessionCard.java:44]: a little unsure on how to convert enum to string ? or is there any other way to set enum? @PotatoCombat

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (PotatoCombat) [reply src/main/java/seedu/address/ui/MainWindow.java:45]: So this variable should match the name given in the fxml file. The @FXML notation helps to link the variable for you.

Comment (PotatoCombat) [reply src/main/java/seedu/address/ui/MainWindow.java:45]: This placeholder is a used to represent an empty but resizable box area in the ui!

StackPane allows you to place things on top of each other.

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 2 above)
Comment (PotatoCombat) [reply src/main/java/seedu/address/ui/MainWindow.java:125]: Here you are placing a ListPanel inside the area designated by the placeholder.

Basically you are "uprooting" the ListPanel, and "planting" it into the placeholder.

PersonListPanel here just takes in an observeable list.

File: src/main/java/seedu/address/ui/SessionCard.java (hunk 3 above)
Comment (PotatoCombat) [reply src/main/java/seedu/address/ui/SessionCard.java:44]: What you are looking for is an Enum Class. You can check out how @fatin99 made it in #120.

Another way is to use a helper function that takes in an enum, selects the correct one using a switch statement, then returns you the corresponding string.

**Pull request closed without merging**